### DISCUSSION TITLE
Blocking right hand scroll bars on websites

### DISCUSSION THREAD

**Discussion by DidntWantToRegister** [uBlockOrigin/uBlock-issues#2501] [Q&A] [answered]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Simply:
Using Chrome latest version, when certain elements are blocked, the right hand scroll bar disappears, and scrolling using space bar, mouse, mousepad etc. is also blocked. 
I cant work out how to resolve this. 
It has happened through multiple browser, UO, and even laptop updates. 
I have searched online for resolutions - many people report it but no fix apparent. 

### A specific URL where the issue occurs.

https://www.glassdoor.co.uk/Reviews/Nuveen-Reviews-E3563.htm

### Steps to Reproduce

1 - go to website

### Expected behavior

Have a scroll bar to allow movement around page

### Actual behavior

no scrollbar. Movement on site blocked. 

### uBO version

1.46.0

### Browser name and version

109.0.5414.120

### Operating System and version

windows 11 latest

**Comment by garry-ut99** [comment]
https://www.reddit.com/r/uBlockOrigin/search/?q=glassdoor When the site shows a register overlay, it disables scroolbar etc, until you accept it/log in, if you want to hide/override it in uBO with cosmetic filters, you also need another filter to unblock the blocked scrollbars etc, either you created bad filters, or you use a 3-rd party filter list which breaks the website.

**Comment by uBlock-user** [comment, marked as answer]
https://www.reddit.com/r/uBlockOrigin/comments/107eie9/unable_to_scroll_down_on/

**Comment by DidntWantToRegister** [comment]
It isn’t resolved. Happens routinely on numerous websites.

**Reply by stephenhawk8054** [reply]
Does the issue occur with uBO default settings (no custom filters and no manual enabled lists)?

**Reply by uBlock-user** [reply]
> It isn’t resolved. Happens routinely on numerous websites.

And you realised this today after 2 yrs ?

**Reply by u-RraaLL** [reply]
> It isn’t resolved.

If a solution is provided and we hear nothing further, the problem will always be considered resolved.

> when certain elements are blocked, the right hand scroll bar disappears, and scrolling using space bar, mouse, mousepad etc. is also blocked.

Do you mean when you block something with element picker/block element?

And if so, then this is precisely what was just referred to as "custom filters".

Anyway, https://www.reddit.com/r/uBlockOrigin/wiki/solutions/#wiki_locked_scroll_after_blocking_various_overlays

**Comment by DidntWantToRegister** [comment]
Yes. I’m a very basic user and have not touched the settings at all. Don’t
know how. It isn’t blocked everywhere but when it is, I’ve found no way of
unblocking.

**Reply by stephenhawk8054** [reply]
Can you open new tab > reproduce the issue and on that page > click on uBO icon > 💬 [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

Can you screenshot how the issue looks like (with the full URL)?

Please reply directly on github. Don't reply via email. Email cannot send images.

**Comment by DidntWantToRegister** [comment]
Will do. Thank you.
